opensrp-anc/src/test/java/org/smartregister/anc/library/fragment/MeFragmentTest.java

```diff
@@ -0,0 +1,21 @@
+package org.smartregister.anc.library.fragment;
+
+import org.junit.Before;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.smartregister.anc.library.AncLibrary;
+import org.smartregister.anc.library.activity.BaseUnitTest;
+
+public class MeFragmentTest extends BaseUnitTest {
+    @Mock
+    private AncLibrary ancLibrary;
+
+    private MeFragment meFragment;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'meFragment' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=5257516)